src/components/organisms/Cart/index.tsx

```diff
@@ -0,0 +1,51 @@
+import XMarkIcon from '@heroicons/react/24/outline/XMarkIcon'
+
+import IconWrapper from 'components/atoms/IconWrapper'
+import Button from 'components/molecules/Button'
+import CartProduct from 'components/molecules/CartProduct'
+import useCartContext from 'contexts/CartContext/useCartContext'
+
+const Cart = () => {
+  const { cart, toggleCart, isOpen, totalPrice, totalItems, clearCart } =
+    useCartContext()
+
+  return (
+    <div
+      className={`${
+        isOpen
+          ? '-translate-x-[0rem] opacity-100'
+          : '-translate-x-[48rem] opacity-0'
+      } flex flex-col justify-between top-0 left-0 fixed z-40 h-full w-[30rem] bg-stone-200 shadow-md md:w-[42rem] overflow-y-auto overflow-x-hidden transition-all duration-500`}
+    >
```

Comment:
this looks terrible, I think that there may be a batter solution which is `classnames` lib